docs/toc_en.yml

```diff
@@ -80,6 +81,7 @@ nav:
     - 'Bit': 'query_language/functions/bit_functions.md'
     - 'Hash': 'query_language/functions/hash_functions.md'
     - 'Generating Pseudo-Random Numbers': 'query_language/functions/random_functions.md'
+    - 'generateUUIDv4': 'query_language/functions/uuid_function.md'
```

Comment:
There are multiple UUID-related functions, so the article should be named like others in plural (uuid_functions.md) with corresponding title.

See:
```
SELECT *
FROM system.functions
WHERE name LIKE '%UUID%'
FORMAT TSV

generateUUIDv4	0	0
toUUID	0	0
UUIDStringToNum	0	0
dictGetUUID	0	0
dictGetUUIDOrDefault	0	0
UUIDNumToString	0	0

6 rows in set. Elapsed: 0.003 sec.
```